### ISSUE TITLE
[Accessibility] Color Contrast is not respected.

### ISSUE THREAD

**Issue by andreacantone** [thunderbird/thunderbird-android#5058]
## Description

The contrast ratio between text and background should be at least 4.5 . Some elements don't respect the contrast ratio. 

## How to reproduce the error

1. Activate the app [Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=it&gl=US)
2. Open the K9 Mail app.
3. Activate the registration from the Accessibility Scanner and use the K9 Mail app.
4. After using the app, stop the recording.

## Screenshot

The results will show the errors found in the app. For example:

![IMG_20201118_164242](https://user-images.githubusercontent.com/25429624/99552899-d2600d00-29bd-11eb-84dc-d7be6c87d7ee.png)

**Comment by wmontwe** [comment]
The account handling has been updated and this is not the case anymore.
